src/components/Tool/TntConnect/uploads/uploadTntConnect.ts

```diff
@@ -45,19 +38,24 @@ export const uploadTnt = async ({
   }
 
   const form = new FormData();
-  form.append('override', override);
-  form.append('tag_list', selectedTags.join(','));
-  form.append('file', file);
-  form.append('accountListId', accountListId);
+  form.append('data[type]', 'imports');
+  form.append('data[attributes][override]', override);
+  form.append('data[attributes][tag_list]', selectedTags.join(','));
+  form.append('data[attributes][file]', file);
 
-  const res = await fetch(`/api/uploads/upload-tnt-connect-import`, {
-    method: 'POST',
-    body: form,
-  }).catch((err) => {
-    throw new Error(t('Cannot upload file: server error') + err);
+  const res = await fetch(
+    `${process.env.REST_API_URL}account_lists/${accountListId}/imports/tnt`,
+    {
+      method: 'POST',
+      headers: {
+        authorization: `Bearer ${apiToken}`,
+      },
+      body: form,
+    },
+  ).catch(() => {
```

Comment:
`fetch` throws if there was a network error such that it couldn't make the request it all. It resolves to a response with `ok: false` if there was an HTTP error (i.e. 4xx or 5xx status code). That's why I switched this error message to be more accurate.